system/core/src/components/CheckboxRadioLabel.ts

```diff
@@ -13,4 +18,9 @@ export const baseStyles = css`
   gap: var(--spacing-l2);
   cursor: pointer;
   font: var(--body-1);
+
+  &[data-variant='disabled'] {
```

Comment:
```suggestion
  &[data-variant='disabled'], &:has(input:disabled) {
```

Let's future proof this which will fix the unchanged checkbox/radio stories.

Reply:
Updated, thanks!